deployment/fabfile.py

```diff
@@ -563,7 +563,7 @@ def install_node_pip(nn='', inst=None):
     pip install matplotlib &&
     pip install gdal==1.10.0 --install-option="build_ext" --install-option="--include-dirs=/usr/include/gdal" &&
```

Comment:
This update is going to have some implications, as Ubuntu 16.04 ships with GDAL 1.11.3: https://launchpad.net/ubuntu/xenial/+source/gdal 

I guess (but I can't be 100% sure) that you'll have to adapt here